heron/ui/src/python/main.py

```diff
@@ -134,14 +136,15 @@ def main():
 
   # log additional information
   command_line_args = vars(parsed_args)
-  address = socket.gethostbyname(socket.gethostname())
+
+  address = '0.0.0.0' if command_line_args['address'] is None else command_line_args['address']
```

Comment:
@kramasamy @objmagic 
In the multiple NICs env socket.gethostbyname(socket.gethostname()) would return virtual NIC address such as vpn, virualbox, docker...
Therefore, I added '0.0.0.0' as Tornado default bind for IPv4, and It's not a good solution too.